README.md

```diff
@@ -14,51 +14,23 @@ _Collaborate and work together on GitHub._
 </header>
 
 <!--
-  <<< Author notes: Course start >>>
-  Include start button, a note about Actions minutes,
-  and tell the learner why they should take the course.
+  <<< Author notes: Step 5 >>>
+  Start this step by acknowledging the previous step.
+  Define terms and link to docs.github.com.
 -->
 
-## Welcome
+## Step 5: Apply suggested changes
 
-All great projects start with collaboration. Pull requests are the foundation of teamwork on GitHub — and pull request reviews give you the ability to work together and discuss changes specific to a pull request by commenting, requesting changes, or approving.
+_Nicely done suggesting changes! :partying_face:_
```

Comment:
```suggestion
_Nicely done suggesting changes! :partying_face:_
```

Very welcoming and fun! Try to include more of this to make it easier on readers.